catroid/src/main/java/org/catrobat/catroid/stage/StageListener.java

```diff
@@ -617,12 +593,13 @@ public void dispose() {
 		disposeClonedSprites();
 	}
 
-	public boolean makeManualScreenshot() {
+	public boolean makeScreenshot(boolean manual) {
```

Comment:
sorry to nitpick here, but would you mind calling it `takeScreenshot(..` instead of `makeScreenshot(..)`, please?

and another idea: how about instead of having a boolean flag you use the sceenshot name as a parameter? then it would be easier to read calls to it `takeScreenshot(ture)` vs. `takeScreenshot(SCREENSHOT_MANUAL_FILE_NAME)`. 

and do you really need the `this` in front of `screenshot`?

Reply:
done.
regarding"this" you're absolutely right, it's not my code, but it's my bad since I forget to delete it.